community-dashboard/app/views/UserGroupDashboard/index.tsx

```diff
@@ -100,6 +104,30 @@ const FILTERED_USER_GROUP_STATS = gql`
     }
 `;
 
+const USER_MEMBERSHIPS_EXPORT = gql`
+    query UserMembershipsExport(
+        $pk: ID!,
+        $limit: Int!,
+        $offset: Int!,
+    ) {
+        userGroup(pk: $pk) {
+            userMemberships(pagination: { limit: $limit, offset: $offset }) {
+                count
+                limit
+                offset
+                items {
+                    userId
+                    username
+                    isActive
+                    totalMappingProjects
+                    totalSwipeTime
+                    totalSwipes
+                }
+            }
+        }
```

Comment:
we need to add `id` for `userGroup`

Reply:
we don't use `id` here, `userId` is used as a unique identifier

Reply:
In that case, we probably need to update apollo cache to ise `userID` as key @tnagorra?

Reply:
We planned to implement this later.
Reference issue: https://github.com/mapswipe/mapswipe/issues/845